.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,165 @@
+name: Build Docker images and push to registry
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+        - main
+    paths:
+        - cmd/**
+        - pkg/**
+        - website/**
+        - Dockerfile
+        - Dockerfile.*
+        - go.mod
+        - go.sum
```

Comment:
The paths are completely wrong, please correct them.
The origin is greenhouse[greenhouse](https://github.com/cloudoperators/greenhouse/blob/9710bcbc12fa3647322753d3b8ccdedd7d6a1060/.github/workflows/docker-build.yaml#L8-L15)